### DISCUSSION TITLE
I am getting a .qmd file in the _site folder of my website

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#10416] [Q&A]
### Description

````qmd
---
title: "Inferencia basada en modelos"
author: Manuel Spínola
institute: ICOMVIS - UNA
format:
  revealjs: 
    theme: [default, mi_tema_02.scss]
    slide-number: true
    footer: "Manuel Spínola"
    self-contained: true
    embed-resources: true
    self-contained-math: true
editor: source
execute:
  echo: false
  warning: false
lang: es
---
````

I am getting a .qmd file in my _site folder.  Any clue why I am getting this behavior?

**Comment by mcanouil** [comment]
Could you properly format your post using code blocks for code and terminal outputs? Thanks.
If your code contains code blocks, you need to enclose it using more backticks, *i.e.*, usually four ` ```` `.
See <https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us>.


Could you share a **small self-contained "working" (reproducible)** example to work with, _i.e._, a complete Quarto document or a Git repository? The goal is to make it as easy as possible for us to recreate your problem so that we can fix it: please help us help you! Thanks.
your code has nothing that can cause that thus it's something else in your environment.

As is, there is nothing to be done to help you unfortunately.

`self-contained` is the old name of `embed-resources`. Why are you setting both?

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Reply by ManuelSpinola** [reply]
Thank you very much.  It was mistake from me in the name of the file. Changing the name solved the problem.
